### DISCUSSION TITLE
Problem with Cascade

### DISCUSSION THREAD

**Discussion by Garuspik24** [vladmandic/sdnext#3002] [Q&A]
Hello. Installed Cascade from Networks/ Reference
Everything is loading without errors
![изображение_2024-03-25_184707220](https://github.com/vladmandic/automatic/assets/164914673/d5502473-428a-4649-a03b-e4f66b6b4153)

but when I try to generate anything I see such errors:
![изображение_2024-03-25_192248629](https://github.com/vladmandic/automatic/assets/164914673/2e1a6a53-9430-4ff1-b660-186c49fcc23b)

This happens at any resolution.
Any suggestions? 
Thanks!

**Comment by vladmandic** [comment]
sc model does not work with float16.
details are in Wiki.

**Reply by Garuspik24** [reply]
Sorry, missed that. Read Wiki. Now cascade make generations but I receive such results : 
![изображение_2024-03-26_171418694](https://github.com/vladmandic/automatic/assets/164914673/1b8e415f-e3c4-48d2-81ac-f99a38a4deac)

**Reply by vladmandic** [reply]
did you adjust other settings (sampler, cfg scale, secondary cfg scale, etc.)?
which sc variant? full or lite?
resolution?
